app/code/core/Mage/Payment/Block/Adminhtml/Payment/Restriction/Grid.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+/**
+ * Maho
+ *
+ * @package    Mage_Payment
+ * @copyright  Copyright (c) 2025 Maho (https://mahocommerce.com)
+ * @license    https://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+
+class Mage_Payment_Block_Adminhtml_Payment_Restriction_Grid extends Mage_Adminhtml_Block_Widget_Grid
+{
+    public function __construct()
+    {
+        parent::__construct();
+        $this->setId('paymentRestrictionGrid');
+        $this->setDefaultSort('restriction_id');
+        $this->setDefaultDir('ASC');
+        $this->setSaveParametersInSession(true);
+    }
+
+    #[\Override]
+    protected function _prepareCollection(): self
+    {
+        $collection = Mage::getModel('payment/restriction')->getCollection();
+        $this->setCollection($collection);
+        return parent::_prepareCollection();
+    }
+
+    #[\Override]
+    protected function _prepareColumns(): self
+    {
+        $this->addColumn('restriction_id', [
+            'header'    => Mage::helper('payment')->__('ID'),
+            'align'     => 'right',
+            'width'     => '50px',
+            'index'     => 'restriction_id',
+        ]);
+
+        $this->addColumn('name', [
+            'header'    => Mage::helper('payment')->__('Name'),
+            'align'     => 'left',
+            'index'     => 'name',
+        ]);
+
+
+        $this->addColumn('status', [
+            'header'    => Mage::helper('payment')->__('Status'),
+            'align'     => 'left',
+            'width'     => '80px',
+            'index'     => 'status',
+            'type'      => 'options',
+            'options'   => [
+                1 => 'Enabled',
+                0 => 'Disabled',
```

Comment:
These aren't translated, but not a huge deal.

Reply:
fixed in https://github.com/MahoCommerce/maho/commit/7530fa64cfae0d606c441cc4f67037fc9ba68c8a